### PR TITLE
Add message support

### PR DESCRIPTION
**Pull request by piecubed**

### CODE REVIEW

**Comment by piecubed** [comment]
The scope of this PR is to add a message database to Fenix.  That includes a way to create and fetch messages by ID, which is fully implemented and passes tests.  It also includes a way to fetch message history, which is surprisingly difficult due to the non RDBMS nature of Cassandra.  I'm not entirely sure the best way to implement this, so suggestions are welcome.

File: client/main.go [hunk 1]
```diff
@@ -28,6 +32,6 @@ func main() {
 	for true {
 		text, _ := reader.ReadString('\n')
 		fmt.Printf("%v%v", ansi.CPL, ansi.DL)
-		c.SendMessage(text)
+		c.SendMessage(sanitize(text))
```
Comment (JosiahWI) [inline client/main.go:35]: Are you sure we don't want to permit newlines?

**Comment by JosiahWI** [comment]
I will test this code thoroughly tomorrow evening and propose a solution to the database issue.

File: client/main.go (hunk 1 above)
Comment (piecubed) [reply client/main.go:35]: Not for the CLI client no.

**Comment by piecubed** [comment]
@JosiahWI I dont think we can really do the same kind of query nicely that i have right now.  Im getting this error: 

```
panic: ORDER BY is only supported when the partition key is restricted by an EQ or an IN.
```
I dont really know if we can have the database get message history easilly, we would have to get a bunch of rows every time and that would be network intensive.  If you have any ideas, that would be great.

**Comment by piecubed** [comment]
@luk3yx You should review this

**Comment by JosiahWI** [comment]
Tested and works great, let's merge.

**Comment by piecubed** [comment]
Let's wait, the test with the latency bug is failing again.

**Review by JosiahWI — DISMISSED** [review]

**Comment by piecubed** [comment]
It should be fine now, you can review it if you want but the tests pass, it should be mergeable!

**Comment by JosiahWI** [comment]
I see all tests passed. Very nice.

**Comment by JosiahWI** [comment]
There are conflicts that need to be resolved; do you want to resolve those now?

**Review by JosiahWI — APPROVED** [review]

**Comment by piecubed** [comment]
Yeah, merging.

**Comment by piecubed** [comment]
Add message support

**Pull request merged**